api/api.yaml

```diff
@@ -18,8 +18,8 @@ paths:
           description: >
             filter represents selected facet values. Elasticsearch query will
             be run only over selected facet values. filter is an array of
-            strings, where each string has the format "facetName=facetValue".
-            Example url /facets?filter=Gender=female,Region=northwest,Region=southwest
+            strings, where each string has the format "facetName=facetValues".
```

Comment:
http://localhost:8390/api/facets?filter=Gender=female,male

didn't work for me. This worked:

http://localhost:8390/api/facets?filter=Gender=female,Gender=male

So the original lines are correct.

After changing api/api.yaml, please either regenerate Swagger or fix generated files.

Reply:
Done